app/src/main/java/org/simple/clinic/facility/alertchange/AlertFacilityChangeSheet.kt

```diff
@@ -30,21 +33,47 @@ class AlertFacilityChangeSheet : BottomSheetActivity() {
   companion object {
     const val FACILITY_CHANGE = 101
     private const val CURRENT_FACILITY_NAME = "current_facility"
+    private const val CONTINUE_TO_SCREEN = "continue_to_screen"
+
+    const val EXTRA_CONTINUE_SCREEN = "extra_continue_screen"
```

Comment:
This need not be public, right?

Reply:
No. It doesn't. I will make this change.